lib/src/browser_client.dart

```diff
@@ -52,16 +52,17 @@ class BrowserClient extends BaseClient {
     request.headers.forEach(xhr.setRequestHeader);
 
     var completer = Completer<StreamedResponse>();
+
+    // TODO(kevmoo): not sure what to do here...
+    // ignore: void_checks
```

Comment:
Here and below – not sure why we're getting this `void_checks` hint...

Reply:
This will be worked around by https://github.com/dart-lang/pedantic/pull/69 if we get that in.

This issue looks like https://github.com/dart-lang/linter/issues/2185

I think we can keep the lint and point the todo at that issue.